public/css/print.css

```diff
@@ -50,14 +52,49 @@ th > * {
 
 td {
 	text-align: center;
+	padding: 18px;
 }
 
 th,
 td {
 	border: 1px solid #a74e94;
 	font-weight: 400;
-	padding: 1.2rem;
-	font-size: 0.8rem;
+	font-size: 15px;
+}
+
+.titles td {
+	padding: 5px;
```

Comment:
Same as the font 15px - loads of padding 5px in this.

I know pdfs are a pain to style - just gets a bit unsettling when everything is laid out in pixels like this. So easy to break - or fall into the trap of pushing stuff around - using styles to unset other styles without realising it.

Not something we need to address now - but maybe in a re-factor one sunny day. :)